### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by TedPogo82**

### CODE REVIEW

**Comment by TedPogo82** [comment]
hello

**Comment by avitacos** [comment]
Hi

File: app.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ document.getElementById("submit").addEventListener("click", function () {
   //get the result
   const result = document.getElementById("result");
 
-  //check conditional
+  //check conditional -
```
Comment (avitacos) [inline app.js:11]: Nice hyphen!

Comment (TedPogo82) [reply app.js:11]: gee thanks

**Review by avitacos — APPROVED** [review]

**Pull request merged**